google/cloud/testing_util/timer.h

```diff
@@ -27,21 +27,29 @@ namespace cloud {
 inline namespace GOOGLE_CLOUD_CPP_NS {
 namespace testing_util {
 
+enum class CpuAccounting {
```

Comment:
Perhaps this is private now.

Reply:
That requires moving `rusage_who()` to the class too, which I thought was a good idea.